extensions/smallrye-fault-tolerance/deployment/src/main/java/io/quarkus/smallrye/faulttolerance/deployment/DotNames.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.smallrye.faulttolerance.deployment;
+
+import java.util.Set;
+
+import org.eclipse.microprofile.faulttolerance.Asynchronous;
+import org.eclipse.microprofile.faulttolerance.Bulkhead;
+import org.eclipse.microprofile.faulttolerance.CircuitBreaker;
+import org.eclipse.microprofile.faulttolerance.Fallback;
+import org.eclipse.microprofile.faulttolerance.Retry;
+import org.eclipse.microprofile.faulttolerance.Timeout;
+import org.jboss.jandex.DotName;
+
+import io.smallrye.common.annotation.Blocking;
+import io.smallrye.common.annotation.NonBlocking;
+import io.smallrye.faulttolerance.api.CircuitBreakerName;
+import io.smallrye.faulttolerance.api.CustomBackoff;
+import io.smallrye.faulttolerance.api.CustomBackoffStrategy;
+import io.smallrye.faulttolerance.api.ExponentialBackoff;
+import io.smallrye.faulttolerance.api.FibonacciBackoff;
+
+public final class DotNames {
+    public static final DotName OBJECT = DotName.createSimple(Object.class.getName());
```

Comment:
Since FT depends on ArC you can also reuse the constants from `io.quarkus.arc.processor.DotNames`.

Reply:
Yea I could. But then I'd use two different classes with the same name, which is not nice :-)

Reply:
Just rename the FT class :-)